skfp/applicability_domain/standard_deviation.py

```diff
@@ -0,0 +1,130 @@
+from numbers import Real
+
+import numpy as np
+from sklearn.utils._param_validation import Interval
+from sklearn.utils.validation import validate_data
+
+from skfp.bases.base_ad_checker import BaseADChecker
+
+
+class StandardDeviationADChecker(BaseADChecker):
+    """
+    Standard deviation method.
+
+    Defines applicability domain based on the spread (standard deviation) of predictions
+    from individual estimators in an ensemble model [1]_. The method assumes that
+    greater consensus among estimators indicates higher prediction reliability,
+    which typically occurs in densely sampled regions of the training data.
+
+    This approach requires an ensemble model exposing the ``estimators_``
+    attribute (e.g., RandomForestRegressor), where each
```

Comment:
Add backticks for model name. Also, I had an idea now that classifiers should also work, as long as they define a `.predict_proba()` method, we should support that. After all, probability estimates also can have standard deviation